src/main/features/offer/form/models/defendantResponse.ts

```diff
@@ -0,0 +1,30 @@
+import { IsDefined, IsIn } from 'class-validator'
+import { Serializable } from 'models/serializable'
+import { StatementType } from './statementType'
+export class ValidationErrors {
+  static readonly OPTION_REQUIRED: string = 'Choose option: yes or no or make an offer'
+}
+
+export class DefendantResponse implements Serializable<DefendantResponse> {
+  @IsDefined({ message: ValidationErrors.OPTION_REQUIRED })
+  @IsIn([StatementType.ACCEPTATION.value, StatementType.REJECTION.value, StatementType.OFFER.value], { message: ValidationErrors.OPTION_REQUIRED })
+  option?: string
```

Comment:
Please add validation unit test similar to `features/response/form/models/rejectPartOfClaim.ts` for example.

Reply:
we already have some router tests which test validation.

Reply:
done

Reply:
Just to clarify route test is not a place to test validation. I roughly test that validation is enabled but not that validation works fine. In route tests we test one scenario but here we have 3 possible options and we need to test all are handled properly.